src/todos/todos.controller.ts

```diff
@@ -1,4 +1,38 @@
-import { Controller } from '@nestjs/common';
+import { Body, Controller, Post, Get, Param, Delete, Put } from '@nestjs/common';
+import { CreateToDoDto } from './dto/create-todo.dto';
+import { TodosService } from './todos.service';
 
 @Controller('todos')
-export class TodosController {}
+export class TodosController {
+    
+    constructor(private todosService: TodosService) {
+
+    }
+
+    @Post()
+    create(@Body() todoDto: CreateToDoDto) {
+        return this.todosService.createToDo(todoDto);
+    }
+
+    @Get()
+    getAll() {
+        return this.todosService.getAllTodos();
+    }
+
+    @Get('/:id')
+    getTodo(@Param('id') id: number) {
+        return this.todosService.getTodo(id);
+    }
+
+    @Put('/:id')
+    updateTodo(@Param('id') id:number, @Body() todoDto: CreateToDoDto) {
+        return this.todosService.updateTodo(id, todoDto)        
+    }
+
+    @Delete('/:id')
+    deleteTodo(@Param('id') id: number) {
+        this.todosService.deleteTodo(id);
+    }
```

Comment:
You need to keep the code in one style. 
You should delete unnecessary empty rows.
And if you're putting the empty row after opening and before closing the scope of the method, you should follow those strategy everywhere.

Reply:
Ok, ty.